boofuzz/blocks/request.py

```diff
@@ -3,9 +3,16 @@
 from .aligned import Aligned
 from .block import Block
 from .. import exception
+from ..exception import BoofuzzNameResolutionError
 from ..fuzzable import Fuzzable
 from ..fuzzable_block import FuzzableBlock
 
+cannot_resolve_name = "Failed to resolve block name '{0}' in context '{1}'"
+name_not_found = "Cannot find block with name '{0}'"
+too_many_matches = (
+    "Unable to resolve block name '{0}'. Use an absolute or relative name instead." " Too many potential matches: {1}"
+)
```

Comment:
We already have some error message constants in `constants.py`. Maybe keep all error messages in one place?
But then again, we'll most likely never use those constants in any other place.
It's up to you.